proto/v1/pagination.proto

```diff
@@ -17,21 +17,23 @@ option go_package = "./proto";
 package datacommons.v1;
 
 
-message ImportGroupCursor {
+// Cursor represents the position of a paginated Bigtable cache data for an
+// import group.
+message Cursor {
   // The index of the import group, starts from 0.
-  int32 import_group = 1;
+  int32 ig = 1;
```

Comment:
Feels 'import_group_index' might be more readable?

Reply:
Okay, reverted back to import_group.